src/platforms/hosted/rtt_if.c

```diff
@@ -66,7 +66,8 @@ int rtt_if_exit()
 
 uint32_t rtt_write(const char *buf, uint32_t len)
 {
-	write(1, buf, len);
+	int l = write(1, buf, len);
```

Comment:
This change suggests that the correct code here should either be `(void)write(1, buf, len);`, or `const int32_t result = write(1, buf, len); (void)result;` - iwrc the former works and feels better. At a minimum, this needs `const` and a non-single-letter variable name using.